### PR TITLE
fix feast repo name in building-and-deploying-multi-stage-RecSys examples

### PR DESCRIPTION
**Pull request by rnyak**
We get this error `AttributeError: module 'feast' has no attribute 'FeatureStore'` from this line `feature_store = feast.FeatureStore(feast_repo_path)` the reason for that is because the name of the subdirectory is feast . And if the working directory is the directory where the feature store was created, then, `import feast`  imports that directory, and not feast library the package.

this PR fixes the feature store repo name, and also removes the version of `feast` that we recommend users to install.

### CODE REVIEW

**Review by nv-alaiacano — APPROVED** [review]
lgtm once tests pass.

File: examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb [hunk 1]
```diff
@@ -75,20 +75,53 @@
     "\n",
     "In case you need to install them for running this example on GPU, execute the following script in a cell.\n",
     "```\n",
-    "%pip install \"feast<0.20\" faiss-gpu\n",
+    "%pip install feast faiss-gpu\n",
```
Comment (jperez999) [inline examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb:78]: This should be anchored to 0.31. Given this is not a 1.0+ version the api may change again.

File: examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb [hunk 2]
```diff
@@ -75,20 +75,53 @@
...
     "```\n",
     "or the following script in a cell for CPU.\n",
     "```\n",
-    "%pip install tensorflow-cpu \"feast<0.20\" faiss-cpu\n",
+    "%pip install tensorflow-cpu feast faiss-cpu\n",
```
Comment (jperez999) [inline examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb:82]: Same as above

File: examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb [hunk 3]
```diff
@@ -129,11 +129,11 @@
    "outputs": [],
    "source": [
     "# for running this example on GPU, install the following libraries\n",
-    "# %pip install \"feast<0.20\" faiss-gpu\n",
+    "# %pip install feast faiss-gpu\n",
```
Comment (jperez999) [inline examples/Building-and-deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:132]: feast should be anchored to 0.31

**Review by jperez999 — DISMISSED** [review]

File: examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb (hunk 1 above)
Comment (rnyak) [reply examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb:78]: thanks for the feedback. let me change that.

**Pull request merged**